.travis.yml

```diff
@@ -21,12 +21,14 @@ addons:
       - librtmp-dev
 install:
   - pip install -U setuptools pip
-  - pip install unittest2 mock pycurl flake8 tox-travis
+  - pip install unittest2 mock pycurl flake8 tox-travis coveralls
   - python setup.py clean --all
   - python setup.py install
 script:
   - if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then flake8 stripe; fi
-  - python -W all -bb -W error::BytesWarning setup.py test
+  - python -W all -bb -W error::BytesWarning -m coverage.__main__ run setup.py test
```

Comment:
I had to use `-m coverage.__main__` instead of `-m coverage` because 2.6 does not support the latter :/

Reply:
Ouch :/ Thanks for explaining anyway.